py-polars/tests/test_series.py

```diff
@@ -278,6 +279,29 @@ def test_set() -> None:
     a = pl.Series("a", [True, False, True])
     mask = pl.Series("msk", [True, False, True])
     a[mask] = False
+    testing.assert_series_equal(a, pl.Series("", [False] * 3))
```

Comment:
@ritchie46 : small detail, but this call drops the `name` from the Series, but other inputs do not result in the same behaviour; see tests below. I would expect the name to stay on updating individual entries.

Reply:
Yes, that is a bug. I will take a look.